### ISSUE TITLE
Message: GetoptLong

### ISSUE THREAD

**Issue by glaubertech** [sbcgua/sap-nw-abap-vagrant#6]
Error in parameter after, for exemplo: vagrant ssh -c "ifconfig"

![image](https://user-images.githubusercontent.com/498927/66007838-4d005d00-e48a-11e9-9cef-ae7754dc888d.png)

![image](https://user-images.githubusercontent.com/498927/66007936-baac8900-e48a-11e9-8f15-5bf72a523847.png)

**Comment by sbcgua** [comment]
will have a look, but a bit busy these days

**Comment by sbcgua** [comment]
should be fixed now, could you try the latest version ?

**Comment by reedtzj** [comment]
![image](https://user-images.githubusercontent.com/5794031/68618858-8e911880-04ca-11ea-8bed-200491c866e5.png)

Hi Alexander

Same problem as above. Tested with latest SP04 and updated github repository.

Sorry for not understanding what caused this error. I hope you can find the time to look into this small problem.

Your Vagrant-solution installing MiniSAP is by far the best I ever seen! 

Regards, Johan von Reedtz

**Comment by sbcgua** [comment]
@reedtzj sorry for so long delay, was really busy. I recreated the issue indeed. It was more complicated than I thought. Seems like vagrant does not allow parsing args in a Vagrant file properly - it conflicts with it's own args. See also [here](https://stackoverflow.com/questions/38234133/how-to-implement-custom-option-for-vagrant-without-having-invalid-option-warning) and [here](https://github.com/hashicorp/vagrant/issues/8718).

So I reworked the way to override VM name to environment variables. Have a look at the  [updated readme section](https://github.com/sbcgua/sap-nw-abap-vagrant#changing-vm-name). And the command from your last message should just works now.

Pls check. and sorry for delay again :)

P.S. I didn't check it really thoroughly so might need some testing ...
